platform/iphone/export/export.cpp

```diff
@@ -156,18 +156,24 @@ struct LoadingScreenInfo {
 };
 
 static const LoadingScreenInfo loading_screen_infos[] = {
-	{ "landscape_launch_screens/iphone_2436x1125", "Default-Landscape-X.png" },
-	{ "landscape_launch_screens/iphone_2208x1242", "Default-Landscape-736h@3x.png" },
-	{ "landscape_launch_screens/ipad_1024x768", "Default-Landscape.png" },
-	{ "landscape_launch_screens/ipad_2048x1536", "Default-Landscape@2x.png" },
-
-	{ "portrait_launch_screens/iphone_640x960", "Default-480h@2x.png" },
-	{ "portrait_launch_screens/iphone_640x1136", "Default-568h@2x.png" },
-	{ "portrait_launch_screens/iphone_750x1334", "Default-667h@2x.png" },
-	{ "portrait_launch_screens/iphone_1125x2436", "Default-Portrait-X.png" },
-	{ "portrait_launch_screens/ipad_768x1024", "Default-Portrait.png" },
-	{ "portrait_launch_screens/ipad_1536x2048", "Default-Portrait@2x.png" },
-	{ "portrait_launch_screens/iphone_1242x2208", "Default-Portrait-736h@3x.png" }
+	// MISSING: Landscape iPhone 5 C S SE
+	// MISSING: Landscape iPhone 6 7 8
+	{ "landscape_launch_screens/iphone_2208x1242", "Default-Landscape-736h@3x.png" }, // iPhone 6 7 8 Plus
+	{ "landscape_launch_screens/iphone_2436x1125", "Default-Landscape-X.png" }, // iPhone X
+	{ "landscape_launch_screens/ipad_1024x768", "Default-Landscape.png" }, // iPad 1 (Likely not needed, way outdated
+	{ "landscape_launch_screens/ipad_2048x1536", "Default-Landscape@2x.png" }, // iPad 2 3 4
+	// MISSING: Landscape iPad Pro 12
+	// MISSING: Landscape iPad Pro 14
+
+	{ "portrait_launch_screens/iphone_640x960", "Default-480h@2x.png" }, // iPhone 4 (Likely not needed, way outdated)
+	{ "portrait_launch_screens/iphone_640x1136", "Default-568h@2x.png" }, // iPhone 5 C S SE
+	{ "portrait_launch_screens/iphone_750x1334", "Default-667h@2x.png" }, // iPhone 6 7 8
+	{ "portrait_launch_screens/iphone_1242x2208", "Default-Portrait-736h@3x.png" }, // iPhone 6 7 8 Plus
+	{ "portrait_launch_screens/iphone_1125x2436", "Default-Portrait-X.png" }, // iPhone X
+	{ "portrait_launch_screens/ipad_768x1024", "Default-Portrait.png" }, // iPad 1 (Likely not needed, way outdated)
+	{ "portrait_launch_screens/ipad_1536x2048", "Default-Portrait@2x.png" }, // iPad 2 3 4
+	// MISSING: iPad Pro 12
+	// MISSING: iPad Pro 14
```

Comment:
When opening the LaunchImage pack in latest Xcode (10.1), it shows all the slots are filled. So how would you fill the missing things there?

![image](https://user-images.githubusercontent.com/2417849/47846082-6079f280-ddf9-11e8-9464-87ebcc4e8018.png)

The project definitely runs on iPhone 6/7/8 showing the launch screen, so I guess it cuts/scales these and it's enough?